x-pack/plugins/apm/public/components/alerting/ui_components/chart_preview/chart_preview_helper.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiLoadingChart } from '@elastic/eui';
+import { EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import moment from 'moment';
+import { i18n } from '@kbn/i18n';
+import { TooltipValue } from '@elastic/charts';
+
+export const tooltipProps = {
+  headerFormatter: (tooltipValue: TooltipValue) =>
+    moment(tooltipValue.value).format('Y-MM-DD HH:mm:ss'),
```

Comment:
@benakansara, Wouldn't it be better to use `uiSettings.get(UI_SETTINGS.DATE_FORMAT)` instead of hardcoded date format?

Reply:
done